### PR TITLE
fix no social link bug, fix no channels bug, styling fixes, linting, …

### PR DESCRIPTION
**Pull request by jlbyrne**
…and smaller bugfixes

### CODE REVIEW

**Comment by thypon** [comment]
gtg

File: nextjs/src/app/[locale]/c/[public_identifier]/QRCodeModal.jsx [hunk 1]
```diff
@@ -37,10 +27,20 @@ export default function QRCodeModal({address, chain, displayChain}) {
...
         });
 
-        qrCode.append(window.document.getElementById('qr-wrapper'));
-      });
+export default function QRCodeModal({address, chain, displayChain}) {
+  const t = useTranslations();
+
+  useEffect(() => {
+    if(typeof window !== 'undefined') {
+      qrCode.append(window.document.getElementById('qr-wrapper'));
```
Comment (porteron) [inline nextjs/src/app/[locale]/c/[public_identifier]/QRCodeModal.jsx:35]: Seems like anti pattern to access the dom directly when using react.

**Review by porteron — APPROVED** [review]
looks solid, left one comment

File: nextjs/src/app/[locale]/c/[public_identifier]/QRCodeModal.jsx (hunk 1 above)
Comment (jlbyrne) [reply nextjs/src/app/[locale]/c/[public_identifier]/QRCodeModal.jsx]: agreed, will fix

**Pull request merged**